src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -3,12 +3,13 @@
 .pf-c-chip-group {
   @extend %pf-t-light; // force the container follow the light theme
 
-  --pf-c-chip-group--PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
-  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip-group--PaddingBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
-  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-chip-group--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip-group--m-toolbar-PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
```

Comment:
if we take off margin top/bottom on the chips you can set this to `var(--pf-global--spacer--xs)` and the same for PaddingBottom.